engine/runtime/src/main/java/org/enso/interpreter/runtime/scope/ModuleScope.java

```diff
@@ -169,7 +202,7 @@ public Function lookupMethodDefinition(Type type, String name) {
         .orElse(null);
   }
 
-  @CompilerDirectives.TruffleBoundary
+  @TruffleBoundary
```

Comment:
Most of the methods in this class have to use `@TruffleBoundary` as their code isn't PE ready at all.

Reply:
Sure, I didn't really change in this regard.